features/bootstrap/HydraContext.php

```diff
@@ -180,9 +180,9 @@ public function assertPropertyIsRequired($propertyName, $className)
      * @param string $propertyName
      * @param string $className
      *
-     * @return array
-     *
      * @throws Exception
+     *
+     * @return array
```

Comment:
Indeed it's better to put `@throws` tag before the `@return`, but we should check that on the whole provide and fix it everywhere. It would be better to do so in another PR as well to avoid polluting this one :)